boa/contracts/vyper/vyper_contract.py

```diff
@@ -26,7 +26,7 @@
 from vyper.codegen.module import generate_ir_for_module
 from vyper.compiler import CompilerData
 from vyper.compiler import output as compiler_output
-from vyper.compiler.output import build_abi_output
+from vyper.compiler.output import build_abi_output, build_solc_json
 from vyper.compiler.settings import OptimizationLevel, anchor_settings
```

Comment:
build_solc_json() throw an invalid path error, as the base path([.]) is not included in the search_paths after the _normalize_path function of the input_bundle is called. This means the resolved_path of the contract's compiler_data file_input is an invalid path.

This error stems from the vyper lib, current trying to fix it. which is why I've not been able to make a PR

Reply:
yeah it works for boa.load (from a file) but not with boa.loads (from a string). See the workaround in the tests.
@charles-cooper is also investigating.

Reply:
Good find